boot/bootstate20.go

```diff
@@ -157,6 +157,8 @@ type bootStateUpdate20 struct {
 
 	// tasks to run after the modeenv has been written
 	postModeenvTasks []bootCommitTask
+
+	revokeOldKeys bool
```

Comment:
not directly related to this PR: but it have been cleaner to add the markSuccessful flag here?